src/model/Global.ts

```diff
@@ -417,6 +422,25 @@ class GlobalModel extends Model<ECUnitOption> {
         return this._theme;
     }
 
+    getLocale(): LocaleOption {
+        return this._locale;
+    }
+
+    getWithLocale(localePosition: Array<string>, optionsPosition?: Array<string>, localeHandlerFn?: (text: string) => string): any {
+        // console.log(optionsPosition, localePosition);
+        const locale = this.getLocale()
+        let localeText: string | any;
+        localePosition.map(t => {
+            localeText = localeText ? localeText[t] : locale[t];
```

Comment:
localePosition can also be wrapped with a Model to do deep query.

For example:
```
const localeModel = this.getLocaleModel();
const localeText = localeModel.get(localePosition);
```

Reply:
Sorry, I didn't quite understand what to do.
Should we add a method as ```getLocaleModel``` for user to get the model of locale?